dotCMS/src/main/java/com/dotcms/api/system/event/local/LocalSystemEventsAPIImpl.java

```diff
@@ -0,0 +1,205 @@
+package com.dotcms.api.system.event.local;
+
+import com.dotcms.concurrent.DotConcurrentFactory;
+import com.dotcms.concurrent.DotSubmitter;
+import com.dotmarketing.util.Logger;
+import com.google.common.annotations.VisibleForTesting;
+
+import java.io.Serializable;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+/**
+ * Default implementation.
+ * The implementation will be provided by {@link LocalSystemEventsAPIFactory}
+ * @author
+ */
+class LocalSystemEventsAPIImpl implements LocalSystemEventsAPI, Serializable {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a "static final long serialVersionUID" field to this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2057)